doc/api/cli.md

```diff
@@ -181,6 +181,17 @@ Make built-in language features like `eval` and `new Function` that generate
 code from strings throw an exception instead. This does not affect the Node.js
 `vm` module.
 
+### `--dns-verbatim=value`
+<!-- YAML
+added: REPLACEME
+-->
+
+Set the default value of `verbatim` in [`dns.lookup()`][]. The value may be:
+* `0` to indicate `false`
+* `1` to indicate `true`
+
+Other values will be ignored.
```

Comment:
I'd make this with no-value (i.e. just `--dns-verbatim` without the `=value`) or an enum (i.e. `--dns-order=verbatim` and `--dns-order=default`)

Reply:
Actually, there are three values: `verbatim`, `ipv4` and `default` so that I think an enum is better, like:
- `--dns-order=verbatim`:  `{verbatim: true}`
- `--dns-order=ipv4`: `{verbatim: false}`
- `--dns-order=default`:  depend on the version of Node.js of which the default values might be different

Might also have a `--dns-order=ipv6` in the future if necessary.